packages/api/src/db/schemas/batchExercise.js

```diff
@@ -8,15 +8,39 @@ const batchExerciseSchema = new Schema({
     minlength: 1,
   },
 
+  day: {
```

Comment:
Assumed you are doing this for efficiency, since you can get this from the linked batchTopic doc

Reply:
Not only for efficiency,
In our batch, everyday exercise from javascript topic included.
To track this functionality, it'll be useful